### PR TITLE
update(MAINTAINERS_GUIDELINES.md): additional (on|off)boarding steps for core maintainers

### PR DESCRIPTION
**Pull request by leogr**
**What type of PR is this?**

> /kind cleanup

/kind documentation

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area governance

> /area proposals

> /area utils

**What this PR does / why we need it**:

Just documenting the current process more precisely. 

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by leogr** [comment]
cc @zuc

File: MAINTAINERS_GUIDELINES.md [hunk 1]
```diff
@@ -60,9 +60,12 @@ New maintainers can also be proposed and sponsored by existing [Maintainers](GOV
...
-- Become a member of the [falcosecurity](https://github.com/falcosecurity) Github organization, see the [Organization embership](#organization-membership) section.
+- If they aren't already, become a member of the [falcosecurity](https://github.com/falcosecurity) Github organization (see the [Organization membership](#organization-membership) section).
 - Join with `*-maintainers` [GitHub team](https://docs.github.com/en/organizations/organizing-members-into-teams/about-teams) relative to the repository they became maintainer of (i.e. the [`falco-maintainers`](https://github.com/orgs/falcosecurity/teams/falco-maintainers) team for [falcosecurity/falco](https://github.com/falcosecurity/falco)); One can do so by opening a PR to change the [org.yaml](https://github.com/falcosecurity/test-infra/blob/master/config/org.yaml) file.
 - Update the [`people/affiliations.json`](people/affiliations.json) file by opening a PR to add their information.
+- Only for first-time core maintainers:
+  - go to https://maintainers.cncf.io/ and open a PR to be listed as a Falco maintainer;
+  - ask to be added to the `cncf-falco-maintainers@lists.cncf.io` mailing list.
```
[the remark this answers is not in the document]
Comment (leogr) [reply MAINTAINERS_GUIDELINES.md:68]: They have to ask the CNCF. Last time I had to write to `cncf-maintainer-changes@cncf.io`, but the correct path could change from time to time, so I would not put it here.

[the remark this answers is not in the document]
Comment (leogr) [reply MAINTAINERS_GUIDELINES.md:68]: What about `ask the CNCF to be added...` ?

[the remark this answers is not in the document]
Comment (leogr) [reply MAINTAINERS_GUIDELINES.md]: Done!

File: MAINTAINERS_GUIDELINES.md [hunk 2]
```diff
@@ -77,11 +80,12 @@ Furthermore, former [Reviewers](GOVERNANCE.md#reviewers) are removed from the [o
...
-
-The person in question must be mentioned in the body of the PR. This acts as a final contact attempt so that they can provide their feedback.
-
-Another PR is required to remove them from GitHub team defined by the [org.yaml](https://github.com/falcosecurity/test-infra/blob/master/config/org.yaml) file.
+In such a case:
+- A PR is required to move the person in question from the `approvers` entry to the `emeritus_approvers` entry of the respective [OWNERS](REPOSITORIES.md#owners) file. The person in question must be mentioned in the body of the PR. This acts as a final contact attempt so that they can provide their feedback.
+- Another PR is required to remove them from GitHub team defined by the [org.yaml](https://github.com/falcosecurity/test-infra/blob/master/config/org.yaml) file.
+- Only for core maintainers who are losing their status:
+  - go to https://maintainers.cncf.io/ and open a PR to remove them under Falco;
+  - ask to remove them from the `cncf-falco-maintainers@lists.cncf.io` mailing list.
```
[the remark this answers is not in the document]
Comment (leogr) [reply MAINTAINERS_GUIDELINES.md]: Done

**Comment by leogr** [comment]
cc @jasondellaluce @maxgio92

**Review by jasondellaluce — APPROVED** [review]
/approve

**Pull request merged**